src/main/java/co/com/bancolombia/utils/Utils.java

```diff
@@ -139,12 +144,35 @@ public static String addModule(String settings, String include, String module, S
     if (settings.contains(toAppend)) {
       return settings;
     }
-    return settings + toAppend;
+    return settings.concat(toAppend);
   }
 
   public static String removeLinesIncludes(String content, String key) {
     return Arrays.stream(content.split("\\n"))
         .filter(line -> !line.contains(key))
         .collect(Collectors.joining("\n"));
   }
+
+  public static String replaceExpression(String content, String regex, String replaceValue) {
+    return content.replaceAll(regex, replaceValue);
+  }
+
+  public static List<String> getAllFilesWithExtension(boolean isKotlin) throws IOException {
+    String extension = isKotlin ? "gradle.kts" : "gradle";
+    List<String> paths;
+    try (Stream<Path> walk = Files.walk(Paths.get("."))) {
```

Comment:
@jakspok that is because in tests this function walk in all directories, and found files that are not found in the project folder, if you have another solution i am open to change that.